homu/main.py

```diff
@@ -1007,11 +1003,18 @@ def queue_handler():
     os.environ['GIT_SSH'] = os.path.join(os.path.dirname(__file__), 'git_helper.py')
     os.environ['GIT_EDITOR'] = 'cat'
 
+    tmp_ssh_key = None
+    if git_cfg['local_git']:
+        tmp_ssh_key = tempfile.NamedTemporaryFile(prefix='homu-sshkey')
```

Comment:
I didn't see any explicit ```close``` of the ```NamedTemporaryFile``` but it still might be worth adding ```delete=False``` to ensure the file isn't deleted if garbage collection decides it's no longer in use.

Example:

```python
import os
import tempfile

def make_temp_file():
    n = tempfile.NamedTemporaryFile()
    n.write(b'ok')
    n.flush()
    return n.name

name = make_temp_file()
# Garbage collection has already closed the file.
print(os.stat(name))
```